### DISCUSSION TITLE
Using chrome extensions with Hero

### DISCUSSION THREAD

**Discussion by vihanga-w** [ulixee/hero#340] [Q&A]
Hi, is there any api that provides support for chrome extensions or would these have to be rewritten as a Hero extension?

**Comment by jensmogens** [comment]
There's no official API for this as of yet, but I've described how you can get extensions to work here: https://github.com/ulixee/hero/issues/266#issuecomment-3517172601

(I am aware that this question is a year old, but I figure it might be helpful for others looking for a solution)
